Model/Tax/Sales/Total/Quote/Tax.php

```diff
@@ -119,6 +119,10 @@ public function collect(
             $quote->getStoreId()
         );
 
+        if (!$isEnabled) {
+            return parent::collect($quote, $shippingAssignment, $total);
+        }
+
         if ($isEnabled) {
```

Comment:
It looks like this `if ($isEnabled)` branch can be removed now.

Reply:
Good catch!